lxd/api_cluster.go

```diff
@@ -3092,14 +3089,68 @@ func evacuateClusterMember(d *Daemon, r *http.Request, mode string) response.Res
 				return err
 			}
 
+			var targetMemberInfo *db.NodeInfo
```

Comment:
Can you split this commit into smaller chunks?

I'd like to cherry-pick at least the variable renames into stable to reduce conflicts in the future.